incubator/demo-ycsb/Chart.yaml

```diff
@@ -12,5 +12,5 @@ maintainers:
   email: support@nuodb.com
 engine: gotpl
 icon: https://raw.githubusercontent.com/nuodb/nuodb-helm-charts/master/images/nuodb.svg
-appVersion: 5.0.4
+appVersion: "5.1"
```

Comment:
The pinned NuoDB image tag seems to be 5.0.1.

Reply:
Good catch.